tests/sles4sap/publiccloud/qesap_prevalidate.pm

```diff
@@ -63,7 +64,14 @@ sub run {
         $self->{my_instance} = $instance;
         last if ($instance->{instance_id} =~ m/vmhana/);
     }
-    $self->wait_for_cluster(wait_time => 60, max_retries => 10);
+
+    # Softfail 12sp5 due to https://bugzilla.suse.com/show_bug.cgi?id=1233026
+    if (is_sle('12+')) {
```

Comment:
`12+` implies 12-SP5 but also `15` and newer. Shouldn't it be `is_sle('=12-SP5')` instead?

Reply:
Thanks, amended! It would work anyway, but by chance